runtime/runtime/src/lib.rs

```diff
@@ -1153,13 +1219,17 @@ impl Runtime {
 
 #[cfg(test)]
 mod tests {
+    use super::*;
+
+    use near::config::INITIAL_GAS_PRICE;
```

Comment:
this is a bit weird to depend on near for just a constant that doesn't matter anyway.

Reply:
It was used within `gas_burnt_to_reward`. So right now it's indirect dependency. I can refactor all of this in a different PR to remove dependency on `near` from `fees_utils`

Reply:
#1701